### PR TITLE
[PBIOS-70] textarea docs

### PR DESCRIPTION
**Pull request by Israel-Molestina**
**What does this PR do?** A clear and concise description with your runway ticket url.
This PR adds a Swift doc example to the playbook webstie
[Runway TextArea Story](https://nitro.powerhrg.com/runway/backlog_items/PBIOS-58)
[Runway Documentation Story](https://nitro.powerhrg.com/runway/backlog_items/PBIOS-70)

**Screenshots:** Screenshots to visualize your addition/change
Screenshot 2023-06-14 at 3 32 12 PM
Screenshot 2023-06-14 at 3 32 23 PM

**How to test?** Steps to confirm the desired behavior:
1. Go to https://pr2603.playbook.beta.px.powerapp.cloud/kits/textarea/swift
2. See swift documentation

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

**Review by jasperfurniss — APPROVED** [review]

File: playbook/app/pb_kits/playbook/pb_textarea/docs/_textarea_swift.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Enables users to add longer text to a form
+
+### When to Use
+For long answers from users
+
+### When not to Use
+For short answers from users, you can use a Text Input instead
+
+### How to use
+| <img width="600" alt="Screenshot 2023-06-14 at 1 50 02 PM" src="https://github.com/powerhome/playbook/assets/73671109/42084be1-84f1-4195-8257-a1cc09bec056"> | <img width="500" alt="2023-06-14 at 2 28 18 PM" src="https://github.com/powerhome/playbook/assets/73671109/8a53e409-5d93-4bc0-b3b2-a40f19588db5"> |
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_textarea/docs/_textarea_swift.md:11]: I'm assuming you added an img tag so that you could tap into the `width`?

Comment (Israel-Molestina) [reply playbook/app/pb_kits/playbook/pb_textarea/docs/_textarea_swift.md:11]: Yes. Otherwise the images are way to big and look weird.

Comment (Israel-Molestina) [reply playbook/app/pb_kits/playbook/pb_textarea/docs/_textarea_swift.md:11]: I still think there is a better way to display the code examples and images so any advice is welcome!

File: playbook/app/pb_kits/playbook/pb_textarea/docs/_textarea_swift.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<img width="300" alt="Screenshot 2023-06-14 at 1 50 02 PM" src="https://github.com/powerhome/playbook/assets/73671109/aeb5750e-6d20-4c3a-8c3c-700c3fe559b9" style="border: 6px solid  black; border-radius: 40px;">
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_textarea/docs/_textarea_swift.md:1]:
```suggestion
![](https://github.com/powerhome/playbook/assets/73671109/aeb5750e-6d20-4c3a-8c3c-700c3fe559b9)
```

Lets keep this standard markdown. We could make it fancier with CSS on the playbook website side. Lets avoid inline styles or custom HTML.

This goes for all the `` tags

**Review by jasoncypret — CHANGES_REQUESTED** [review]

**Pull request closed without merging**